changelog/issue-4943.md

```diff
@@ -0,0 +1,6 @@
+audience: users
+level: patch
+reference: issue 4943
+---
+
+Fixes UI error "buildSignedUrl missing required credentials" shown directly after login due to missing credentials.
```

Comment:
Do you know when this bug was introduced? Could we add a note to the change log to explicitly say which release introduced this bug, so users know if they have it and need to update or not?

Reply:
I think it was there for 5+ years. Only happens when user session changes. Hard do tell if that feature caused it or some other auth related one did